app/src/main/res/layout/vlc_player_interface.xml

```diff
@@ -91,15 +91,37 @@
             android:layout_marginTop="20sp"
             android:layout_marginEnd="40sp" />
 
-        <ImageView
-            android:id="@+id/logoImage"
-            android:layout_width="190sp"
-            android:layout_height="60sp"
+        <LinearLayout
```

Comment:
"LinearLayout-3 can overlap @id/textClock if LinearLayout-3 grows due to localized text expansion"

This might cause issues, make sure the width is limited (by adding the clock to this linearlayout or using a constraintlayout or something)